main.go

```diff
@@ -14,8 +17,16 @@ func main() {
 	mw := MeasurementsWriter{measurements: measurements, writeAPI: client.WriteAPIBlocking("tsm", "mydb")}
 	mw.AsyncStart()
 
+	m, err := mongo.Connect(context.Background(), options.Client().ApplyURI(os.Getenv("TSM_MONGO_URI")))
+	if err != nil {
+		panic(err)
+	}
+
 	myRouter := mux.NewRouter().StrictSlash(true)
-	deviceService := DeviceService{dao: &inMemoryDeviceDAO{}}
+
+	devices := m.Database("tsm").Collection("devices")
```

Comment:
We usually pass database, not collection. it's up to dao to pick collection. of course, in 99% cases it's gonna be one collection, but it happens

Reply:
One more reason to pass db instead of collection: this is `main` file. DB is usually shared between different DAOs, but collection rarely is.